galaxy/templates/configmap-galaxy-conf.yaml

```diff
@@ -0,0 +1,17 @@
+{{- $fullName := include "galaxy.fullname" . -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ $fullName }}-config-map
+  labels:
+    app: {{ template "galaxy.name" . }}
+    chart: {{ template "galaxy.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+data:
+  {{- range $key, $entry := .Values.configs }}
+  {{ $key }}: |
+    {{- if $entry.contents -}}
```

Comment:
Would this work if entry.contents is None? Because the previous line starts a multi-line string, and then there's no line afterwards?